docs/manual/manual_angle_control_mechanisms.rst

```diff
@@ -0,0 +1,647 @@
+    module.set("multi_turn_angle_control", "angular_speed_max", 25)
+
+    #Set the control displacement to 0 rad
+    module.set("multi_turn_angle_control", "ctrl_angle", 0)
+
+    #Give it time to settle
+    time.sleep(3)
+
+    #Set the control displacement to 0 rad
+    module.set("multi_turn_angle_control", "ctrl_coast")
+
+Trajectory and Maximum Angular Speed
+-------------------------------------------------------
+
+In this next example, we explore how *Maximum Angular Speed* affects trajectories. First, we reset the module to 0 rad of displacement. 
+Then, we set the *angular_speed_max* to 10 rad/s. A reminder that this is the absolute fastest that the module will ever spin when commanded. 
+The trajectory that we command is a displacement of 100 rad over 5 seconds. Normally, this results in an average speed of about 20 rad/s 
+with a slower start and stop, and faster maximum speed. With the *Maximum Angular Speed* set to 10 rad/s, however, the trajectory will take 
+about 10 seconds to complete at a speed capped at 10 rad/s. 
+
+.. code-block:: python
+    :linenos:
+
+    import iqmotion as iq
+    import time
+
+
+    #Create a SerialCommunicator object for our servo module
+    com = iq.SerialCommunicator("COM3")
+    module = iq.ServoModule(com)
+
+
+    #Make sure we're reset to 0 displacement
+    module.set("multi_turn_angle_control", "angular_speed_max", 50)
+    module.set("multi_turn_angle_control", "ctrl_angle", 0)
+    time.sleep(2)
+
+
+    #Set the maximum angular speed to 10 rad/s
+    module.set("multi_turn_angle_control", "angular_speed_max", 10)
+
+
+    #Send a trajectory that should take about 5 seconds at around 20 rad/s
+    module.set("multi_turn_angle_control", "trajectory_angular_displacement", 100)
+    module.set("multi_turn_angle_control", "trajectory_duration", 5)
+
+Notice that this trajectory completes in about 10 seconds rather than the expected 5. This is because the angular velocity is limited to 10 rad/s 
+rather than the calculated 20 rad/s.
+
+Notice that the module does not reach its destination until 10 seconds. The trajectory itself, however, completes in the commanded 5 seconds. 
+If you queue another trajectory, the original commanded trajectory will carry into the next trajectory. Trajectories still execute in the background as 
+originally commanded, and are not modified by these limits.
+
+Zero Angle
+++++++++++++++++++++++++++++
+The module's *Zero Angle* is the physical position to be treated as 0 radians of displacement 
+(assuming no explicit sets of *Observed Angular Displacement*. See :ref:`Setting Observed Displacement and its Relationship with Zero Angle`). 
+*Zero Angle* is a value bounded by [-:math:`\pi`, :math:`\pi`], giving a range of the entire circle around the module. Displacement is the 
+total distance (radial or linear) traveled away from *Zero Angle*. Displacement is not bounded by [-:math:`\pi`, :math:`\pi`], and can be any 32-bit float 
```

Comment:
Might want to say "floating point" value instead of "float". Float is just shorthand for that